api/management/swagger-api_management.yaml

```diff
@@ -968,6 +968,77 @@ paths:
           description: successful operation
         400:
           description: invalid information provided  (invalid client identifier or redirect URI not allowed for this client)
+  /realms/{realm}/backoffice-configuration:
+    get:
+      tags:
+      - Configuration
+      summary: Get the current backoffice configuration. Uses groups of the current user.
```

Comment:
this one uses also the groups of the user? the two endpoints do the same thing?

Reply:
I updated the description... One "get" method is relative to the current user (its realm and its groups)... The two other methods are used to get and set the back-office configuration for a given realmID/groupName

Reply:
Please document what happens if a user has multiple groups :-)